src/app/loadout-drawer/LoadoutDrawer2.tsx

```diff
@@ -234,13 +235,16 @@ export default function LoadoutDrawer2({
       <LoadoutDrawerHeader loadout={loadout} onNameChanged={handleNameChanged} />
       <details className={styles.notes} open={Boolean(loadout.notes?.length)}>
         <summary>{t('MovePopup.Notes')}</summary>
-        <TextareaAutosize
-          onChange={handleNotesChanged}
-          ref={ref}
-          value={loadout.notes}
-          maxLength={2048}
-          placeholder={t('Loadouts.NotesPlaceholder')}
-        />
+        <div style={{ display: 'flex' }}>
```

Comment:
Maybe we need a component that wraps a textarea plus this button?

Reply:
Yeah, should be better this way.